```
/_/src/Components/Components/src/Routing/RouteTableFactory.cs(113,17): Trim analysis error IL2067: Microsoft.AspNetCore.Components.RouteTableFactory.Create(Dictionary<Type,String[]>): 'handler' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.Routing.RouteEntry.RouteEntry(RouteTemplate, Type, List<String>)'. The parameter 'key' of method 'System.Collections.Generic.KeyValuePair<TKey,TValue>.Deconstruct(TKey&, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Components/src/Routing/RouteTableFactory.cs(113,17): error IL2067: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.RouteTableFactory.Create(Dictionary<Type,String[]>): 'handler' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.Routing.RouteEntry.RouteEntry(RouteTemplate, Type, List<String>)'. The parameter 'key' of method 'System.Collections.Generic.KeyValuePair<TKey,TValue>.Deconstruct(TKey&, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to.
/_/src/Components/Components/src/Routing/RouteTableFactory.cs(113,17): Trim analysis error IL2067: Microsoft.AspNetCore.Components.RouteTableFactory.Create(Dictionary<Type,String[]>): 'handler' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.Routing.RouteEntry.RouteEntry(RouteTemplate, Type, List<String>)'. The parameter 'key' of method 'System.Collections.Generic.KeyValuePair<TKey,TValue>.Deconstruct(TKey&, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Components/src/Routing/RouteTableFactory.cs(113,17): error IL2067: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.RouteTableFactory.Create(Dictionary<Type,String[]>): 'handler' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.Routing.RouteEntry.RouteEntry(RouteTemplate, Type, List<String>)'. The parameter 'key' of method 'System.Collections.Generic.KeyValuePair<TKey,TValue>.Deconstruct(TKey&, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to.
/_/src/Components/Components/src/RouteView.cs(82,9): Trim analysis error IL2118: Microsoft.AspNetCore.Components.RouteView.Render(RenderTreeBuilder): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Components/src/RouteView.cs(82,9): error IL2118: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.RouteView.Render(RenderTreeBuilder): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member.
/_/src/Components/Components/src/RouteView.cs(82,9): Trim analysis error IL2118: Microsoft.AspNetCore.Components.RouteView.Render(RenderTreeBuilder): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Components/src/RouteView.cs(82,9): error IL2118: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.RouteView.Render(RenderTreeBuilder): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member.
/_/src/Components/Authorization/src/AuthorizeRouteView.cs(104,9): Trim analysis error IL2118: Microsoft.AspNetCore.Components.Authorization.AuthorizeRouteView.RenderContentInDefaultLayout(RenderTreeBuilder, RenderFragment): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Authorization/src/AuthorizeRouteView.cs(104,9): error IL2118: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.Authorization.AuthorizeRouteView.RenderContentInDefaultLayout(RenderTreeBuilder, RenderFragment): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member.
/_/src/Components/Authorization/src/AuthorizeRouteView.cs(104,9): Trim analysis error IL2118: Microsoft.AspNetCore.Components.Authorization.AuthorizeRouteView.RenderContentInDefaultLayout(RenderTreeBuilder, RenderFragment): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Authorization/src/AuthorizeRouteView.cs(104,9): error IL2118: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.Authorization.AuthorizeRouteView.RenderContentInDefaultLayout(RenderTreeBuilder, RenderFragment): Compiler-generated member 'Microsoft.AspNetCore.Components.LayoutView.<>c__DisplayClass13_0.<WrapInLayout>g__Render|0(RenderTreeBuilder)' is accessed via reflection. Trimmer can't guarantee availability of the requirements of the member.
/_/src/Components/Web/src/JSComponents/JSComponentInterop.cs(73,9): Trim analysis error IL2067: Microsoft.AspNetCore.Components.Web.Infrastructure.JSComponentInterop.AddRootComponent(String, String): 'componentType' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.RenderTree.WebRenderer.AddRootComponent(Type, String)'. The parameter 'value' of method 'System.Collections.Generic.Dictionary<TKey,TValue>.TryGetValue(Dictionary<TKey,TValue>.TKey, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Web/src/JSComponents/JSComponentInterop.cs(73,9): error IL2067: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.Web.Infrastructure.JSComponentInterop.AddRootComponent(String, String): 'componentType' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.RenderTree.WebRenderer.AddRootComponent(Type, String)'. The parameter 'value' of method 'System.Collections.Generic.Dictionary<TKey,TValue>.TryGetValue(Dictionary<TKey,TValue>.TKey, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to.
/_/src/Components/Web/src/JSComponents/JSComponentInterop.cs(73,9): Trim analysis error IL2067: Microsoft.AspNetCore.Components.Web.Infrastructure.JSComponentInterop.AddRootComponent(String, String): 'componentType' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.RenderTree.WebRenderer.AddRootComponent(Type, String)'. The parameter 'value' of method 'System.Collections.Generic.Dictionary<TKey,TValue>.TryGetValue(Dictionary<TKey,TValue>.TKey, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to. [/Users/runner/work/1/s/src/Components/WebAssembly/testassets/WasmLinkerTest/WasmLinkerTest.csproj]
##[error]/_/src/Components/Web/src/JSComponents/JSComponentInterop.cs(73,9): error IL2067: (NETCORE_ENGINEERING_TELEMETRY=Build) Microsoft.AspNetCore.Components.Web.Infrastructure.JSComponentInterop.AddRootComponent(String, String): 'componentType' argument does not satisfy 'DynamicallyAccessedMemberTypes.All' in call to 'Microsoft.AspNetCore.Components.RenderTree.WebRenderer.AddRootComponent(Type, String)'. The parameter 'value' of method 'System.Collections.Generic.Dictionary<TKey,TValue>.TryGetValue(Dictionary<TKey,TValue>.TKey, TValue&)' does not have matching annotations. The source value must declare at least the same requirements as those declared on the target location it is assigned to.
  HealthChecksSample -> /Users/runner/work/1/s/artifacts/bin/HealthChecksSample/Release/net7.0/HealthChecksSample.dll
  LinkabilityChecker -> /Users/runner/work/1/s/artifacts/bin/LinkabilityChecker/Release/net7.0/LinkabilityChecker.dll
  Microsoft.AspNetCore.SignalR.Client.Tests -> /Users/runner/work/1/s/artifacts/bin/Microsoft.AspNetCore.SignalR.Client.Tests/Release/net7.0/Microsoft.AspNetCore.SignalR.Client.Tests.dll
  Microsoft.AspNetCore.Identity.Test -> /Users/runner/work/1/s/artifacts/bin/Microsoft.AspNetCore.Identity.Test/Release/net7.0/Microsoft.AspNetCore.Identity.Test.dll
```
https://dev.azure.com/dnceng/public/_build/results?buildId=1828841&view=logs&j=f43b187f-6c15-531c-a59b-e4fba4e9ed20&t=98a09275-a4dd-593e-c21a-1f8bea3be5b1&l=2975

There are new linker warnings in components that need to be fixed.

cc @mkArtakMSFT @SteveSandersonMS @MackinnonBuck

--- a/global.json
+++ b/global.json
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.5.22263.22"
+    "version": "7.0.100-preview.5.22272.3"
   },
   "tools": {
-    "dotnet": "7.0.100-preview.5.22263.22",
+    "dotnet": "7.0.100-preview.5.22272.3",
     "runtimes": {
       "dotnet/x86": [
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"

--- a/global.json
+++ b/global.json
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.6.22301.14"
+    "version": "7.0.100-preview.6.22306.16"
   },
   "tools": {
-    "dotnet": "7.0.100-preview.6.22301.14",
+    "dotnet": "7.0.100-preview.6.22306.16",
     "runtimes": {
       "dotnet/x86": [
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"

--- a/global.json
+++ b/global.json
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "7.0.100-preview.5.22274.1"
+    "version": "7.0.100-preview.5.22273.27"
   },
   "tools": {
-    "dotnet": "7.0.100-preview.5.22274.1",
+    "dotnet": "7.0.100-preview.5.22273.27",
     "runtimes": {
       "dotnet/x86": [
         "$(MicrosoftNETCoreBrowserDebugHostTransportVersion)"

